source/code/plugins/auditd_plugin_lib.rb

```diff
@@ -0,0 +1,57 @@
+require 'yajl'
```

Comment:
Please add unittests. I could not find any. you can look into other ruby plugin unittests that use mock objects for testing.